docs/tutorials/evpn-mh/basics/verify.md

```diff
@@ -0,0 +1,461 @@
+    13:58:10.862696 IP 192.168.0.11.49747 > 192.168.0.21.199: Flags [S], seq 2625981132, win 1024, options [mss 1460], length 0
+    13:58:10.862795 IP 192.168.0.11.49747 > 192.168.0.23.111: Flags [S], seq 2625981132, win 1024, options [mss 1460], length 0
+    ```
+=== "eth2 tcpdump"
+    ```text linenums="1" hl_lines="12"
+    bash-5.0# tcpdump -ni eth2
+    tcpdump: verbose output suppressed, use -v or -vv for full protocol decode
+    listening on eth2, link-type EN10MB (Ethernet), capture size 262144 bytes
+    13:58:10.810595 ARP, Reply 192.168.0.21 is-at 00:c1:ab:00:00:21, length 28
+    13:58:10.857768 IP 192.168.0.11.49747 > 192.168.0.22.445: Flags [S], seq 2625981132, win 1024, options [mss 1460], length 0
+    13:58:10.857916 IP 192.168.0.11.49747 > 192.168.0.22.23: Flags [S], seq 2625981132, win 1024, options [mss 1460], length 0
+    13:58:10.857962 IP 192.168.0.11.49747 > 192.168.0.22.3389: Flags [S], seq 2625981132, win 1024, options [mss 1460], length 0
+    13:58:10.858002 IP 192.168.0.11.49747 > 192.168.0.22.443: Flags [S], seq 2625981132, win 1024, options [mss 1460], length 0
+    13:58:10.859723 IP 192.168.0.22.445 > 192.168.0.11.49747: Flags [R.], seq 0, ack 2625981133, win 0, length 0
+    13:58:10.859846 IP 192.168.0.22.3389 > 192.168.0.11.49747: Flags [R.], seq 0, ack 2625981133, win 0, length 0
+    13:58:10.862474 IP 192.168.0.11.49747 > 192.168.0.22.199: Flags [S], seq 2625981132, win 1024, options [mss 1460], length 0
+    13:58:10.862478 IP 192.168.0.23.445 > 192.168.0.11.49747: Flags [R.], seq 0, ack 2625981133, win 0, length 0
+    13:58:10.862592 IP 192.168.0.21.445 > 192.168.0.11.49747: Flags [R.], seq 0, ack 2625981133, win 0, length 0
+    13:58:10.862620 IP 192.168.0.11.49747 > 192.168.0.22.111: Flags [S], seq 2625981132, win 1024, options [mss 1460], length 0
+    13:58:10.862653 IP 192.168.0.23.3389 > 192.168.0.11.49747: Flags [R.], seq 0, ack 2625981133, win 0, length 0
+    13:58:10.862727 IP 192.168.0.11.49747 > 192.168.0.22.143: Flags [S], seq 2625981132, win 1024, options [mss 1460], length 0
+    13:58:10.862772 IP 192.168.0.21.3389 > 192.168.0.11.49747: Flags [R.], seq 0, ack 2625981133, win 0, length 0
+    13:58:10.866605 IP 192.168.0.11.49747 > 192.168.0.22.80: Flags [S], seq 2625981132, win 1024, options [mss 1460], length 0
+    ```
+
+Check the tcpdump output for `eth1` and `eth2` interfaces when sending traffic with nmap. You should see that the traffic is balanced for both incoming and outgoing packets.  
+For example, the highlighted line in `eth1` output shows the outgoing request `IP 192.168.0.11.49747 > 192.168.0.23.445: Flags [S]` going out from `eth1` interface. And then the reply `IP 192.168.0.23.445 > 192.168.0.11.49747: Flags [R.]` is seen on `eth2`.
+
+!!!note
+    Outgoing ARP packets may not be balanced because load balancing mode of the bond is layer2 by default.
+
+## EVPN Routes
```

Comment:
@aaakpinar 
I think what is not quite covered here is how leaf3 load balances towards leaf1/leaf2 for traffic reaching CE.

This is an important part in understanding the multihoming part. E.g. which ESI is going to be used